app/elements/secrets-init.html

```diff
@@ -239,7 +240,8 @@
 				},
 				_watchPolicyRequests: function() {
 					// Wait for all policy requests to return. Once complete, trigger _parseAccess()
-					if (this.loginResponse && this.loginResponse.policies && this.policyRequests.length == this.loginResponse.policies.length) {
+					var policies = this.loginResponse.policies.concat(this.loginResponse.identity_policies);
+					if (this.loginResponse && this.loginResponse.policies && this.policyRequests.length == policies.length) {
```

Comment:
This checks for the existence of values prior to using them. The creation of a new variable `policies` happens prior to this check.